build-system/tasks/jsify-css.js

```diff
@@ -45,6 +45,9 @@ cssnano = cssnano({
   mergeIdents: true,
   reduceIdents: false,
   zindex: false,
+  svgo: {
+    encode: true,
```

Comment:
huh, kind of surprised  `postcss-svgo` detected that they were already encoded. Regardless I think we should decode all SVGs we have for consistency. Shouldn't be too much work (its like 2 or 3 that are fully encoded. The rest just encoded `#`). You can maybe set `svgo: { encode: false }`, do a run and take the decoded versions from the build.